src/main/java/com/stripe/model/PaymentIntent.java

```diff
@@ -0,0 +1,285 @@
+package com.stripe.model;
+
+import com.stripe.exception.APIConnectionException;
+import com.stripe.exception.APIException;
+import com.stripe.exception.AuthenticationException;
+import com.stripe.exception.CardException;
+import com.stripe.exception.InvalidRequestException;
+import com.stripe.net.APIResource;
+import com.stripe.net.RequestOptions;
+
+import java.util.List;
+import java.util.Map;
+
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class PaymentIntent extends APIResource implements MetadataStore<PaymentIntent>, HasId {
+  @Getter(onMethod = @__({@Override})) String id;
+  String object;
+  List<String> allowedSourceTypes;
+  Long amount;
+  Long amountCapturable;
+  Long amountReceived;
+  @Getter(AccessLevel.NONE) @Setter(AccessLevel.NONE) ExpandableField<Application> application;
+  Long applicationFee;
+  Long canceledAt;
+  String captureMethod;
+  ChargeCollection charges;
+  String clientSecret;
+  String confirmationMethod;
+  Long created;
+  String currency;
+  @Getter(AccessLevel.NONE) @Setter(AccessLevel.NONE) ExpandableField<Customer> customer;
+  Boolean livemode;
+  @Getter(onMethod = @__({@Override})) Map<String, String> metadata;
+  PaymentIntentSourceAction nextSourceAction;
+  @Getter(AccessLevel.NONE) @Setter(AccessLevel.NONE) ExpandableField<Account> onBehalfOf;
+  String receiptEmail;
+  String returnUrl;
```

Comment:
Can you confirm it's the right way to write url and that it does not need anything custom?